eclair-rpc-test/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcClientTest.scala

```diff
@@ -742,6 +679,120 @@ class EclairRpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
 
   }
 
+  it should "get a route to an invoice" in {
+    val hasRoute = () => {
+      fourthClientF.flatMap { fourthClient =>
+        fourthClient
+          .receive("foo")
+          .flatMap { invoice =>
+            firstClientF.flatMap(_.findRoute(invoice, None))
+          }
+          .map(route => route.length == 4)
+          .recover {
+            case err: RuntimeException
+                if err.getMessage.contains("route not found") =>
+              false
+          }
+      }
+
+    }
+
+    // Eclair is a bit slow in propagating channel changes
+    AsyncUtil.awaitConditionF(hasRoute, duration = 10000.millis, maxTries = 20).map(_ => succeed)
+  }
+
+
+*/
+
+  it should "update the relay fee of a channel" in {
+    val channelAndFeeF = for {
+      channel <- openAndConfirmChannel(clientF, otherClientF)
+      feeOpt <- clientF.flatMap(_.channel(channel).map(_.feeBaseMsat))
+    } yield {
+      assert(feeOpt.isDefined)
+      assert(feeOpt.get > MilliSatoshis.zero)
+      (channel, feeOpt.get)
+    }
+
+    for {
+      (channel, oldFee) <- channelAndFeeF
+      _ <- clientF.flatMap(
+        _.updateRelayFee(channel, MilliSatoshis(oldFee.toLong * 2), 1))
+      newFeeOpt <- clientF.flatMap(_.channel(channel).map(_.feeBaseMsat))
+    } yield {
+      assert(newFeeOpt.isDefined)
+      assert(newFeeOpt.get == MilliSatoshis(oldFee.toLong * 2))
+    }
+  }
+
+  it should "update the relay fee of a channel with short channel id" in {
+    val channelAndFeeF = for {
+      channelId <- openAndConfirmChannel(clientF, otherClientF)
+      client <- clientF
+      channel <- client.channel(channelId)
+    } yield {
+      assert(channel.feeBaseMsat.isDefined)
+      assert(channel.feeBaseMsat.get > MilliSatoshis.zero)
+      (channel.channelId, channel.shortChannelId, channel.feeBaseMsat.get)
+    }
+
+    for {
+      (channelId, shortChannelId, oldFee) <- channelAndFeeF
+      _ <- clientF.flatMap(
+        _.updateRelayFee(shortChannelId, MilliSatoshis(oldFee.toLong * 4), 1))
```

Comment:
Can just do `oldFee * 4`, instead of `(shortChannelId, MilliSatoshis(oldFee.toLong * 4)`